include/deal.II/lac/parpack_solver.h

```diff
@@ -913,7 +913,7 @@ void PArpackSolver<VectorType>::solve
           AssertThrow (false, PArpackExcInfoPdneupd(info));
         }
 
-      for (size_type i=0; i<nev; ++i)
+      for (size_type i=0; i<(size_type)nev; ++i)
```

Comment:
`int nev = n_eigenvalues`, which is done to (i) match fortran definition (ii) make `nev` modifiable, which to my understanding, could happen for non-symmetric systems. So it's probably either 
```
for (int i=0; i<nev; ++i)
```
or what's currently in the patch.